### PR TITLE
Fix the dependencies in the quickstart

### PR DESCRIPTION
**Pull request by timyates**
They were broken as reactor: is an asciidoc macro

### CODE REVIEW

File: src/main/docs/guide/quickStart.adoc [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Add the following dependency to your Micronaut application:
 
-dependency:io.micronaut.reactor:micronaut-reactor[scope="compile", version="{version}"]
+dependency:micronaut-reactor[scope="compile", groupId="io.micronaut.reactor", version="{version}"]
```
Comment (sdelamo) [inline src/main/docs/guide/quickStart.adoc]: this generates 

```
implementation("io.micronaut.reactor:micronaut-reactor:2.2.1-SNAPSHOT")
```

I think what we should have is: 

```
implementation("io.micronaut.reactor:micronaut-reactor")
```

which we could achieve with: 

```
dependency:micronaut-reactor[groupId="io.micronaut.reactor"]
```

File: src/main/docs/guide/quickStart.adoc [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 To use the HTTP client add the following dependency:
 
-dependency:io.micronaut.reactor:micronaut-reactor-http-client[scope="compile", version="{version}"]
+dependency:micronaut-reactor-http-client[scope="compile", groupId="io.micronaut.reactor", version="{version}"]
```
Comment (sdelamo) [inline src/main/docs/guide/quickStart.adoc]: this generates 

```
implementation("io.micronaut.reactor:micronaut-reactor-http-client:2.2.1-SNAPSHOT")
```

I think what we should have is: 

```
implementation("io.micronaut.reactor:micronaut-reactor-http-client")
```

which we could achieve with: 

```
dependency:micronaut-reactor-http-client[groupId="io.micronaut.reactor"]
```

**Review by sdelamo — APPROVED** [review]
I have done some changes to resolve the conversations I created. I hope you are ok with those changes.

File: src/main/docs/guide/quickStart.adoc (hunk 2 above)
Comment (timyates) [reply src/main/docs/guide/quickStart.adoc]: @sdelamo Good catch...  I manually updated the docs https://micronaut-projects.github.io/micronaut-reactor/latest/guide/ which shows 2.2.0 (as I was on the released commit when I generated them).  Do you think we need to manually publish again?  Or is this ok to wait until next release?

**Review by sdelamo — APPROVED** [review]

File: src/main/docs/guide/quickStart.adoc (hunk 2 above)
Comment (sdelamo) [reply src/main/docs/guide/quickStart.adoc]: I think it is ok to wait.

**Pull request merged**